docs/development/bpmn-support-how-to.md

```diff
@@ -39,6 +39,18 @@ Refer to existing Pull Requests to have a better view about the work to do, for
 - [Manual Task Detection Pull Request](https://github.com/process-analytics/bpmn-visualization-js/pull/582/files)
 
 
+### BPMN model update
+
+When changing the model, the image representing the model in the documentation needs to be updated as well.
+
+The source of the image is stored as a [draw.io file](../architecture/images/architecture/internal-model.drawio).
+
+To update this file
+- edit it with https://app.diagrams.net or with the `diagrams.net` desktop application
+- store the modified file in the repository 
```

Comment:
I think we need to put the name of the file explicitly.

At the first reading, I didn't understand why you want to store it 2 times. But after I remember we need to store the .drawio file and the .svg file ^^